Data/src/main/java/org/tribuo/data/columnar/RowProcessor.java

```diff
@@ -85,6 +85,9 @@
     @Config(description="A map from a regex to field processors to apply to fields matching the regex.")
     protected Map<String,FieldProcessor> regexMappingProcessors = new HashMap<>();
 
+    @Config(description="Replace newlines with spaces in values before passing them to field processors.")
+    protected boolean replaceNewlinesWithSpaces;
```

Comment:
This needs to be set to `true` otherwise old configs will load with it set to `false` and break config compatibility.